providers/ipmitool/ipmitool.go

```diff
@@ -58,7 +58,7 @@ func (c *Conn) Compatible(ctx context.Context) bool {
 		return false
 	}
 	defer c.Close(ctx)
-	_, err = c.con.IsOn(ctx)
+	_, err = c.con.PowerState(ctx)
```

Comment:
yep, these showed up after the master rebase - since the change has the sha 062ef08b, I believe git should handle it (as in it won't be another commit)? .. unless I'm missing something